### DISCUSSION TITLE
color in print_message

### DISCUSSION THREAD

**Discussion by LPeccoud** [cwapi3d/cwapi3dpython#155] [Q&A]
HI
How can i colorize print_message.
ANSI Escape sequences work in in my Windows 10 command prompt but not inCadwork.
Thanks for you answers

**Comment by roehligas** [comment]
I don't have an answer, I'm just curious.

You talk about the function that prints into the Cadwork-window, not the python console right?

**Reply by LPeccoud** [reply]
Yes. In cadwork window

**Comment by jspaquet** [comment]
Unfortunately, if you're talking about print_message and print_error we don't support custom colors. Colors in Cadwork 3D have a very specific meaning (e.g. green for keyboard shortcuts) and in order for this to remain consistent for users, plugins need to follow the same UI guidelines as the rest of the interface.

**Reply by LPeccoud** [reply]
Ok, thanks.
But how to do that (keyboard shortcuts)  in print_message string.
